### PR TITLE
fixed active filters for rails 4.2

### LINKED ISSUE #4342: No current filters when use filters in sidebar
When i use filters i see **no current filters** in sidebar.

### PR DESCRIPTION
**Pull request by Fivell**
[fixes #4342]

### CODE REVIEW

File: lib/active_admin/filters/active.rb [hunk 1]
```diff
@@ -8,13 +8,21 @@ class Active
...
-        @params = params.respond_to?(:to_unsafe_h) ? params.to_unsafe_h : params
+        @params = normalize_params(params)
         @scope = humanize_scope
         @filters = build_filters
       end
 
       private
 
+      def normalize_params(params)
+        if params.kind_of?(HashWithIndifferentAccess)
```
Comment (deivid-rodriguez) [inline lib/active_admin/filters/active.rb]: There's much more usages of `is_a?` than `kind_of?` in the codebase (35 vs 3 if I counted corrected), so maybe use `is_a?` to keep it more consistent.

**Comment by deivid-rodriguez** [comment]
This makes sense to me, just added a very minor comment.

File: lib/active_admin/filters/active.rb (hunk 1 above)
Comment (varyonic) [inline lib/active_admin/filters/active.rb]: `HashWithIndifferentAccess === params ? params : params.to_unsafe_h` should work also.

Comment (Fivell) [inline lib/active_admin/filters/active.rb]: @varyonic , it doesn't for rails 4.2

Loading development environment (Rails 4.2.7.1)
2.2.0 :001 > params = ActionController::Parameters.new(name: 'Test')
 => {"name"=>"Test"} 
2.2.0 :002 > params === HashWithIndifferentAccess
 => false

Comment (Fivell) [inline lib/active_admin/filters/active.rb]: in rails 4.2 ActionController::Parameters inherited from HashWithIndifferentAccess, it even have to_unsafe_h method, but it works not that way like in rails5

**Pull request merged**